resource-aggregate/cqrs/aggregate/aggregate.go

```diff
@@ -198,6 +198,15 @@ func (a *Aggregate) handleCommandWithAggrModel(ctx context.Context, cmd Command,
 		}
 		return events, false, nil
 	}
+
+	if len(newEvents)+amodel.numEvents >= a.numEventsInSnapshot {
```

Comment:
Could you please explain this change?

Reply:
one line below. In our use case, if you have 100 pending commands and then you cancel them, it's generated 100 events. But the threshold of creating a snapshot is set to 16, so to optimize the next reading we add the snapshot as the last event.